src/snyk/snykCode/views/suggestion/codeSuggestionWebviewProvider.ts

```diff
@@ -110,8 +110,27 @@ export class CodeSuggestionWebviewProvider
         this.registerListeners();
       }
 
+      this.panel.iconPath = vscode.Uri.joinPath(
+        vscode.Uri.file(this.context.extensionPath),
+        'media',
+        'images',
+        'snyk-code.svg',
+      );
+
       if (isIgnoresEnabled) {
-        this.panel.webview.html = issue.additionalData.details;
+        let html = issue.additionalData.details;
+        const ideStylePath = vscode.Uri.joinPath(
```

Comment:
Instead of linking the stylesheet like we do for other panels we are injecting the styling into the `<style></style>` element that has the nonce. The reason for this is to be consistent with how this panel is built in IntelliJ, where JCEF is not able to find local CSS files. I could refactor this code to inject the stylesheet entirely to be consistent with other HTML panels but then we will need to also generate a new nonce for the CSP. Not sure what the best option here is, but I chose what I thought was quicker and made more sense for now